### ISSUE TITLE
Index.html override causes router to use hash

### ISSUE THREAD

**Issue by Kaplaugher** [gridsome/gridsome#818]
## Description

It seems when I override index.html it causes the router to change to the hash mode. 

### Steps to reproduce

Create a new index.html in source and add external script. In this case I added snipcart. 
![code](https://user-images.githubusercontent.com/22328417/68998625-92ca8680-087a-11ea-8c32-0dfdc369804d.png)

### Expected result

Router would continue with history mode as it does by default.

### Actual result

Router switches to hash and I am unable to change the mode.

### Environment

Paste the information here as shown by `gridsome info`
 System:
    OS: Windows 10
    CPU: (12) x64 Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
  Binaries:
    Node: 10.16.0 - C:\Program Files\nodejs\node.EXE
    npm: 6.10.1 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: 44.17763.831.0
  npmPackages:
    gridsome: 0.7.0 => 0.7.0

**Comment by hjvedvik** [comment]
Does this happen with all scripts you import or just when importing the Snipcart theme?

**Comment by Kaplaugher** [comment]
Yes, it seems it is just snipcart. Others I have tested work fine.

**Comment by HARAJLI98** [comment]
I'm having the same issue described. Any updates on what's going on?
